### ISSUE TITLE
use of unexplained constants in chgres_cube code search_util.f90

### ISSUE THREAD

**Issue by edwardhartnett** [NOAA-EMC/UFS_UTILS#278]
In chgres_cure.fd/search_util.f90 I see:

```
subroutine sst_guess(latitude, sst)

 use esmf

 implicit none

 real(esmf_kind_r8), intent(in)  :: latitude

 real(esmf_kind_r8), intent(out) :: sst

 if (latitude >= 60.0) then
   sst = 273.16
 elseif (abs(latitude) <= 30.0) then
   sst = 300.0
 else
   sst = (-0.8947) * abs(latitude) + 326.84
 endif

 end subroutine sst_guess
```

We have a bunch of undocumented constants here. ;-)

These should be defined as constants and properly documented.

**Comment by edwardhartnett** [comment]
Any progress here?

**Comment by GeorgeGayno-NOAA** [comment]
I think there is a bug in the first 'if' block.  I suspect it should be:

`if (abs(latitude) > 60.0) then`

**Comment by edwardhartnett** [comment]
OK, good find. Can you please enter a new issue for that?

**Comment by GeorgeGayno-NOAA** [comment]
> OK, good find. Can you please enter a new issue for that?

This dovetails nicely with #436, because the unit test I wrote exercises this routine. Once #436 is merged, I will update the unit test for this specific case.
